### DISCUSSION TITLE
How to use it with  Vue

### DISCUSSION THREAD

**Discussion by ZhaoXH123** [mathuo/dockview#608] [Q&A]
I install dockview-vue:
    npm install dockview-vue

but i I cannot find a complete example。
this is  my code：
```ts
<template>
  <DockviewVue @ready="onReady">
  </DockviewVue>
</template>

<script setup lang="ts">

import type { DockviewApi, DockviewReadyEvent, DockviewVue } from 'dockview-vue';

const onReady = (event: DockviewReadyEvent) => {
  const api: DockviewApi = event.api;
}
</script>

<style scoped></style>
```

but "dockview-vue" file  is:  import * from "dockview-cor".  don't have DockviewVue.

**Comment by Walraz** [comment]
You have declared your import as types. Try import { DockviewVue } from ”dockview-vue”

**Comment by memtech3** [comment]
#757 looks like a dupe of this with more diagnostic information
